vividus-plugin-browserstack/build.gradle

```diff
@@ -4,10 +4,8 @@ dependencies {
     implementation project(':vividus-engine')
     implementation project(':vividus-util')
     implementation project(':vividus-reporter')
-    implementation project(':vividus-soft-assert')
     implementation project(':vividus-extension-selenium')
-    implementation project(':vividus-reporter')
```

Comment:
it's actually used: https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-browserstack/src/main/java/org/vividus/browserstack/steps/DebugSteps.java#L44

Reply:
It's duplicated see the line 6